model/lockStrategy/SqlStorage.php

```
@@ -68,57 +70,67 @@ public static function getPersistence()
     }
-    
-    public static function add($userId, core_kernel_classes_Resource $resource, core_kernel_classes_Resource $copy) {
+
+    public static function add($userId, core_kernel_classes_Resource $resource, core_kernel_classes_Resource $copy)
+    {
         $persistence = self::getPersistence();
-		$query = 'INSERT INTO "'.self::TABLE_NAME.'" ("'.self::FIELD_OWNER.'", "'.self::FIELD_RESOURCE.'", "'.self::FIELD_WORKCOPY.'", "'.self::FIELD_CREATED.'") VALUES (?,?,?,?)';
-		$result = $persistence->exec($query,array($userId, $resource->getUri(), $copy->getUri(), time()));
+        $query = 'INSERT INTO "' . self::TABLE_NAME . '" ("' . self::FIELD_OWNER . '", "'
+            . self::FIELD_RESOURCE . '", "' . self::FIELD_WORKCOPY . '", "' . self::FIELD_CREATED . '") VALUES (?,?,?,?)';
+        $result = $persistence->exec($query, [$userId, $resource->getUri(), $copy->getUri(), time()]);
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$result'. (undefined)

(UnusedLocalVariable)